README.adoc

```diff
@@ -877,6 +877,16 @@ a|Number of pixels to move the parallax background per slide
 |<a valid CSS display mode>
 |The display mode that will be used to show slides.
 Defaults to *block*
+
+|:revealjs_pdfSeparateFragments:
```

Comment:
You need to use lowercase:

```suggestion
|:revealjs_pdfseparatefragments:
```


See: https://github.com/asciidoctor/asciidoctor-reveal.js/commit/134bf3d29b82db787db069e30290144463e4f033

Reply:
There are already revealjs_parallaxBackgroundHorizontal, revealjs_showNotes, etc. most options are revealjs_camelCaseOption. The only exceptions are revealjs_minscale and revealjs_maxscale. It doesn't make sense to me lowercase the option.
Are you sure you want me to change this?

Reply:
> There are already revealjs_parallaxBackgroundHorizontal, revealjs_showNotes, etc. most options are revealjs_camelCaseOption. The only exceptions are revealjs_minscale and revealjs_maxscale. It doesn't make sense to me lowercase the option.

It has changed recently, now all the attributes are lowercased (on master).

> Are you sure you want me to change this?

Yes.

> Although uppercase characters are permitted in an attribute entry (the place where an attribute is defined), the attribute name is converted to lowercase before being stored. The attribute name in an attribute reference is also converted to lowercase before the attribute is resolved. For example, URI, Uri and uRI are all treated as uri. (See issue #509 for a proposed change to this restriction). A best practice is to only use lowercase for letters in the name and avoid starting the name with a number.

https://asciidoctor.org/docs/user-manual/#attribute-restrictions

In short, you can define `:revealjs_camelCaseOption:` in your document but the attribute name will be converted to `revealjs_camelcaseoption`. So in the template we must use lowercase.
To avoid confusion, we also use lowercase in the documentation.

Reply:
@Mogztter On the branch master, it's still camel case, there's something I don't understand? Your link is pointing to asciidoctor manual, not asciidoctor-reveal.js.

Reply:
Do you have a link where it's still in camel case? Here's what I get on the master branch: 

https://github.com/asciidoctor/asciidoctor-reveal.js/blob/45a02329209d5a48a2ba6932102376189f35c222/templates/document.html.slim#L128-L244

> Your link is pointing to asciidoctor manual, not asciidoctor-reveal.js.

Yes but Asciidoctor Reveal.js is using Asciidoctor to resolve attributes so we must comply with the above rule.

Reply:
Here https://raw.githubusercontent.com/asciidoctor/asciidoctor-reveal.js/master/README.adoc

Reply:
Oh I see... technically it works because `revealjs_camelCaseOption` will be resolved to `revealjs_camelcaseoption`.

@obilodeau Do you have an opinion? Should we also use lowercase in the documentation?

Reply:
Yeah, at first I created a commit that fixed the case for all the doc but then I realized: why bother? It worked for people before and it will continue working. If attributes ever become case-sensitive then we will fix it and in the meantime it is more readable than without casing.